ddtrace/context.py

```diff
@@ -241,3 +240,39 @@ def get(self):
             self._locals.context = ctx
 
         return ctx
+
+
+class _ContextVarContext(object):
```

Comment:
Why is this private?

Reply:
We don't anticipate this is something anyone should be using and also allows us to rework it in the future. I can certainly see the benefit to keeping `Context` in the public API, though the contextvars and the thread-local storage might be best pulled out into the internal submodule.